src/Subscriber/AbstractStatusChangeSubscriber.php

```diff
@@ -13,6 +13,7 @@ abstract class AbstractStatusChangeSubscriber implements EventSubscriberInterfac
         'needs work' => Status::NEEDS_WORK,
         'works for me' => Status::WORKS_FOR_ME,
         'reviewed' => Status::REVIEWED,
+        'waiting_feedback' => Status::WAITING_FEEDBACK,
```

Comment:
```suggestion
        'waiting feedback' => Status::WAITING_FEEDBACK,
```

Reading the code in `parseStatusFromText()` I would have expected something like this instead as I don't think anyone will write "Status: waiting_feedback" in a PR comment.

Reply:
true!